### PR TITLE
Framework

### PR DESCRIPTION
**Pull request by jaskooo**
Changed xcode target from static lib to framework to reduce problems with builds

### CODE REVIEW

File: syncano-ios.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -489,32 +494,52 @@
 				C6C26CD1570D29893B2F95D7 /* libPods-syncano4-iosTests.a */,
```
Comment (lifcio) [inline syncano-ios.xcodeproj/project.pbxproj]: Something old? Can be removed?

File: syncano-ios/Syncano.h [hunk 2]
```diff
@@ -23,6 +25,14 @@
 #import "SCChannel.h"
 #import "SCFile.h"
 
+//! Project version number for Syncano.
+FOUNDATION_EXPORT double SyncanoVersionNumber;
+
+//! Project version string for Syncano.
+FOUNDATION_EXPORT const unsigned char SyncanoVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <Syncano/PublicHeader.h>
```
Comment (lifcio) [inline syncano-ios/Syncano.h]: Looking at the comment - why are we doing `#import "SCFile.h"` instead? Either comment or our import is wrong.

**Comment by lifcio** [comment]
Not sure what happened here exactly - did we change the name of our Pod? Is it `Syncano` instead of `syncano-ios` now?

File: syncano-ios/Syncano.h (hunk 2 above)
Comment (jaskooo) [inline syncano-ios/Syncano.h]: No. This change is only in xcode project. I have changed target from static lib to framework. It should not affect Pods

File: syncano-ios.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jaskooo) [inline syncano-ios.xcodeproj/project.pbxproj]: Sure :)

**Pull request merged**